src/Helpers/RecognizeClassHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+use Illuminate\Support\Str;
+
+class RecognizeClassHelper
+{
+    public static function recognizeObjectClass(string $objectName): string
+    {
+        if (strpos($objectName, "\\")) {
+            return $objectName;
+        }
+
+        $objectName = Str::ucfirst(Str::singular($objectName));
+
+        return self::getObjectNamespace($objectName) . $objectName;
+    }
+
+    public static function getObjectNamespace(string $objectName): string
+    {
+        $type = Str::plural(self::guessType($objectName));
+
+        return "App\\$type\\";
+    }
+
+    public static function guessType(string $objectName): string
+    {
+        foreach (TypesEnum::cases() as $objectType) {
+            $slug = Str::slug($objectName);
```

Comment:
This should be run before foreach to minimalise number of unnecessary slugging actions.